test/Microsoft.DotNet.ProjectModel.Tests/LockFilePatchingTests.cs

```diff
@@ -77,6 +77,15 @@ public void TestMissingExportFileThrows()
 
             Assert.Throws<FileFormatException>(() => LockFileReader.Read(lockFilePath, designTime: false));
         }
+        
+        [Fact]
+        public void TestMissingExportUnderDesignTime()
+        {
+            var lockFilePath = GetLockFilePath("invalid_nofragment");
+
+            // not throw under design time scenario
+            Assert.NotNull(LockFileReader.Read(lockFilePath, designTime: true));            
```

Comment:
Can you also please assert the the actual Lock file is not the empty one? Exceptions originating from the LockFileReader are swallowed and a mock lock file is returned. Patching exceptions though should not cause this mock lock file to be returned.